net/https-dns-proxy/files/README.md

```diff
@@ -1,5 +1,8 @@
+<!-- markdownlint-disable MD013 -->
 # DNS Over HTTPS Proxy (https-dns-proxy)
 
+[![HitCount](http://hits.dwyl.com/stangri/openwrt/https-dns-proxy.svg)](http://hits.dwyl.com/stangri/openwrt/https-dns-proxy)
```

Comment:
No way. Sorry. NACK for this change.

Also the CDN which you are using is questionable as we don't have such things under control.

Reply:
As far as CDN goes, are you talking about screenshots at jsdelivr? Please elaborate how is linking private repo on github better than using jsdelivr.

Reply:
README has been moved out of OpenWrt repo.